lib/ElementHandle.js

```diff
@@ -121,9 +119,14 @@ class ElementHandle extends JSHandle {
    * @return {!Promise<?{x: number, y: number, width: number, height: number}>}
    */
   async boundingBox() {
-    const result = await this._client.send('DOM.getBoxModel', {
-      objectId: this._remoteObject.objectId
-    }).catch(error => void debugError(error));
+    let result = null;
```

Comment:
I'm not sure I understand why this has changed. Is it necessary?

Reply:
I think it ended up at this after a few rounds of changes, almost full circle. Will complete the circle.

Reply:
Fixed